src/routes/stats/components/StatsLeaderboardCell.vue

```diff
@@ -9,7 +9,6 @@
           ...props,
           ...dataAttribute,
         }"
-        dark
```

Comment:
This is no longer supported and I don't know of another prop with a similar result. 🤷🏼

Reply:
I'm not sure this actually matters since we don't have a dark/light theme